prometheus/base/alerting/prometheus.pod_alerts.yaml

```
@@ -25,3 +25,11 @@ spec:
       annotations:
         summary: "Pod {{ $labels.pod }} is crash looping"
         description: "Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is in waiting state (reason: 'CrashLoopBackOff') for more than 15 minutes."
+    - alert: PodsNotReady
+      expr: kube_pod_status_phase{phase=~"Pending|Unknown|Failed", namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"} == 1 unless ignoring (phase) (kube_pod_status_unschedulable == 1)
```

Can we break this line a bit for readability? Something like this:
```suggestion
      expr: |
            kube_pod_status_phase{phase=~"Pending|Unknown|Failed", namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"} == 1
            unless ignoring (phase) (kube_pod_status_unschedulable == 1)
```